### PR TITLE
BC-312 - added new DefaultWireframe to task overview

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

Adding the new default layout to task overview

## Links to Ticket and related Pull-Requests

- https://ticketsystem.hpi-schul-cloud.org/browse/BC-312

## Screenshots of UI changes

Screenshot 2021-09-29 at 19 25 18
Screenshot 2021-09-29 at 19 25 01

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

**Comment by odalys-dataport** [comment]
@schul-cloud-bot update screenshots

File: src/components/templates/DefaultWireframe.vue [hunk 1]
```diff
@@ -4,10 +4,15 @@
 			v-if="breadcrumbs.length"
 			:breadcrumbs="breadcrumbs"
 		></v-custom-breadcrumbs>
+		<div
+			v-else
+			:class="{ 'breadcrumbs-placeholder': $vuetify.breakpoint.smAndUp }"
```
Comment (pkleybolte) [inline src/components/templates/DefaultWireframe.vue:9]: I would prefer to use a static class and put the media breakpoint in the CSS

File: src/components/templates/DefaultWireframe.vue [hunk 2]
```diff
@@ -4,10 +4,15 @@
...
+		></div>
 		<slot name="header">
 			<h1 class="text-h3">
 				{{ headline }}
 			</h1>
+			<div class="border"></div>
```
Comment (pkleybolte) [inline src/components/templates/DefaultWireframe.vue:15]: If I understood @monaweitzenberg correct, then we should put the border outside of the slot. So we always have the lien between the header and the content section

Comment (odalys-dataport) [reply src/components/templates/DefaultWireframe.vue]: Good point, done!

**Review by CeEv — APPROVED** [review]

**Comment by odalys-dataport** [comment]
@schul-cloud-bot update screenshots

**Comment by odalys-dataport** [comment]
@schul-cloud-bot update screenshots

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**